metricflow/model/data_warehouse_model_validator.py

```diff
@@ -139,3 +166,13 @@ def validate_data_sources(self, model: UserConfiguredModel, timeout: Optional[in
         """
         tasks = DataWarehouseTaskBuilder.gen_data_source_tasks(model=model)
         return self.run_tasks(tasks=tasks, timeout=timeout)
+
+    def validate_metrics(self, model: UserConfiguredModel, timeout: Optional[int] = None) -> List[ValidationIssue]:
+        """Generates a list of tasks for validating the metrics of the model and then runs them
+
+        :param model UserConfiguredModel: Model which to run data warehouse validations on
```

Comment:
Ahhh gotcha. I think I started using this pattern because of how things were commented in `base_sql_client_implementation.py` when I was writing the `dry_run` method. Looks like we might have some cleanup to do sometime to get `element_const.py`, `sql_client.py`, and `base_sql_client_implementation.py` comments converted sometime.